Modules/StudyProgramme/classes/class.ilPrgUpdateProgressCronJob.php

```diff
@@ -71,13 +69,24 @@ public function getDefaultScheduleValue()
     public function run()
     {
         $result = new ilCronJobResult();
-        foreach ($this->user_progress_db->getPassedDeadline() as $progress) {
+        $result->setStatus(ilCronJobResult::STATUS_NO_ACTION);
+        $acting_user = $this->getActingUserId();
+        foreach ($this->getProgressRepository()->readPassedDeadline() as $progress) {
```

Comment:
```suggestion
        foreach ($this->getProgressRepository()->getPassedDeadline() as $progress) {
```
Just `read` -> `get` again...